impeller/renderer/backend/vulkan/surface_vk.cc

```diff
@@ -12,12 +12,18 @@ namespace impeller {
 
 std::unique_ptr<SurfaceVK> SurfaceVK::WrapSwapchainImage(
     const std::shared_ptr<Context>& context,
-    const std::shared_ptr<SwapchainImageVK>& swapchain_image,
+    std::shared_ptr<SwapchainImageVK>& swapchain_image,
     SwapCallback swap_callback) {
   if (!context || !swapchain_image || !swap_callback) {
     return nullptr;
   }
 
+  // Some Vulkan devices may not support memoryless (lazily allocated) textures.
+  // In this case we will cache the MSAA texture on the swapchain image to avoid
+  // thrasing the VMA heap.
+  bool supports_memoryless =
+      context->GetCapabilities()->SupportsMemorylessTextures();
```

Comment:
I think you lost part of your PR in one of your pushes.  `SupportsMemorylessTextures` isn't in the PR anymore.

Reply:
that was on a prior PR that already landed.